webapp/views/App/views/Home/Dashboard/ActivityLog/store/parsers/utils.js

```diff
@@ -1,28 +1,27 @@
 import * as R from 'ramda'
+import * as A from '@core/arena'
 
 import * as ActivityLog from '@common/activityLog/activityLog'
 import * as Survey from '@core/survey/survey'
 
+const mergePropsAndPropsDraft = (item) =>
+  A.isEmpty(item) ? item : { ...item, props: { ...(item.props || {}), ...(item.propsDraft || {}) } }
```

Comment:
this must be done server side. there's a common method that does that.
DB.transformCallback